xbmc/addons/kodi-addon-dev-kit/doxygen/Skin/skin.dox

```diff
@@ -0,0 +1,27 @@
+/*!
+
+\page skin_parts Skin Development
+\brief \htmlonly
+      <h3><span  style="text-decoration: underline;"><span  style="font-style: italic;"><span
+               style="color: rgb(102, 102, 102);">Skin Add-On Development</span></span></span></h3>
+       \endhtmlonly
+
+Kodi includes a GUI library that allows you to skin/change everything you see in
```

Comment:
Just a suggestion to break up your long sentences: Kodi includes a GUI library that allows you to skin/change everything you see. The images, sizes and positions of controls, colours, fonts, text, through to altering navigation and even adding new functionality.